### ISSUE TITLE
Raunkiaer's classifications

### ISSUE THREAD

**Issue by siobhangorham** [Canadensys/vascan-data#2661]
I collected Raunkiaer classification information a few years ago for a project and I believe I collected it from VASCAN.  Did you provide that information in the past?

Thank you,
Siobhan

**Comment by brouille** [comment]
Siobhan

We never included information on Raunkiaer’s classification in VASCAN. You may have taken it from a dagtabase on ecological traits?

Luc

De : siobhangorham <notifications@github.com>
Répondre à : Canadensys/vascan-data <reply@reply.github.com>
Date : mercredi 12 avril 2017 14:31
À : Canadensys/vascan-data <vascan-data@noreply.github.com>
Cc : Subscribed <subscribed@noreply.github.com>
Objet : [Canadensys/vascan-data] Raunkiaer's classifications (#2661)


I collected Raunkiaer classification information a few years ago for a project and I believe I collected it from VASCAN. Did you provide that information in the past?

Thank you,
Siobhan

**Comment by siobhangorham** [comment]
Merci Luc!

I must have collected it from another site.  I went back through my notes
and found a page where I noted Scoggan's Flora of Canada as having the
classifications but I can't seem to retrace my steps.  I found the Scoggan
paper but it doesn't have the classifications.

Thank you for getting back to me.

Cheers,
Siobhan

**Comment by brouille** [comment]
Scoggan’s 1978-79 flora provides a summary of life forms. I do not know whether it provides data for each taxon (which would not all match VASCAN).

Luc

De : siobhangorham <notifications@github.com>
Répondre à : Canadensys/vascan-data <reply@reply.github.com>
Date : mardi 18 avril 2017 11:02
À : Canadensys/vascan-data <vascan-data@noreply.github.com>
Cc : Luc Brouillet <luc.brouillet@umontreal.ca>, Comment <comment@noreply.github.com>
Objet : Re: [Canadensys/vascan-data] Raunkiaer's classifications (#2661)

Merci Luc!

I must have collected it from another site. I went back through my notes
and found a page where I noted Scoggan's Flora of Canada as having the
classifications but I can't seem to retrace my steps. I found the Scoggan
paper but it doesn't have the classifications.

Thank you for getting back to me.

Cheers,
Siobhan

**Comment by MFavreau** [comment]
Yes, Scoggan does provide life-form info for each species treated in his flora. Symbols in brackets, e.g. (Hs), refer to table 3, p. 21 of vol. 1.

Of course, the species are as per 1978 taxonomy, at best...

Marc Favreau

(a tabula)

-------- Message d'origine --------
De :brouille <notifications@github.com>
Envoyé :Tue, 18 Apr 2017 12:55:57 -0400
À :Canadensys/vascan-data <vascan-data@noreply.github.com>
Ci :Subscribed <subscribed@noreply.github.com>
Sujet :Re: [Canadensys/vascan-data] Raunkiaer's classifications (#2661)

>Scoggan’s 1978-79 flora provides a summary of life forms. I do not know whether it provides data for each taxon (which would not all match VASCAN). 
>
>Luc 
>
>De : siobhangorham <notifications@github.com> 
>Répondre à : Canadensys/vascan-data <reply@reply.github.com> 
>Date : mardi 18 avril 2017 11:02 
>À : Canadensys/vascan-data <vascan-data@noreply.github.com> 
>Cc : Luc Brouillet <luc.brouillet@umontreal.ca>, Comment <comment@noreply.github.com> 
>Objet : Re: [Canadensys/vascan-data] Raunkiaer's classifications (#2661) 
>
>Merci Luc! 
>
>I must have collected it from another site. I went back through my notes 
>and found a page where I noted Scoggan's Flora of Canada as having the 
>classifications but I can't seem to retrace my steps. I found the Scoggan 
>paper but it doesn't have the classifications. 
>
>Thank you for getting back to me. 
>
>Cheers, 
>Siobhan 
>

**Comment by siobhangorham** [comment]
Yes, thank you.  I believe that's what I used and why some of the
classifications don't agree with some current literature.  Unfortunately I
am no longer in Quebec and don't have close access to the full volume.  But
the current literature does have some classifications and I've been able to
update my database.

I really appreciate your time.  It's a somewhat esoteric system but I think
it addresses the life history of Arctic plants in a useful way.

Regards,
Siobhan
